src/expression.d

```diff
@@ -1564,10 +1585,11 @@ extern (C++) bool functionParameters(Loc loc, Scope* sc, TypeFunction tf, Type t
             }
             else
             {
-                //                arg = arg->isLvalue() ? callCpCtor(sc, arg) : valueNoDtor(arg);
+                //arg = doCopyOrMove(sc, arg);
```

Comment:
To be fair, the comment should not have been starting at column zero, so it was already badly indented before ddmd.